stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpUtil.java

```diff
@@ -735,6 +736,20 @@ public static ChunkConfig getChunkConfig(String chunking) {
         return chunkConfig;
     }
 
+    public static ForwardedExtensionConfig getForwardedExtensionConfig(String forwarded) {
+        ForwardedExtensionConfig forwardedConfig;
```

Comment:
Wouldn't an enum be a better fit for this? Like we've done in chunking and keep-alive configs?

Reply:
Yes, ForwardedExtensionConfig is an Enum.